lib/fog/compute/google/models/network.rb

```diff
@@ -36,12 +36,10 @@ def destroy(async = true)
           operation
         end
 
-        # Returns a ready API structure for insert_instance, used in insert_server request.
-        def get_as_interface_config(access_config = nil)
```

Comment:
Thanks for the context. Finally I have left this method as it was (please see changes in the source code). By the way according to [Google API documentation](https://cloud.google.com/compute/docs/reference/beta/instances) "if the network is not specified but the subnetwork is specified, the network is inferred". It means in the case when a user doesn't mention a network, it should not to be defaulted but to be inferred from a subnetwork. I find this a little bit strange.